src/components/usePopup.ts

```diff
@@ -15,6 +15,7 @@ export type PopupProps = {
     setCountryPercentage: (country: CountryName, percentage: number) => void;
     averageSpecificEmissions: number;
     refreshAndGetSize: (bypassCache: boolean) => Promise<void>;
+    stopRecording: () => void;
```

Comment:
how come this is in your props when you've defined it in the component?

Reply:
Good catch!